test/WebJobs.Script.Tests/Configuration/ExtensionBundleOptionsSetupTest.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.BindingExtensionBundle;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ExtensionBundleOptionsSetupTest
+    {
+        private readonly TestEnvironment _environment = new TestEnvironment();
+        private readonly TestLoggerProvider _loggerProvider = new TestLoggerProvider();
+        private readonly string _hostJsonFile;
+        private readonly ScriptApplicationHostOptions _options;
+
+        public ExtensionBundleOptionsSetupTest()
+        {
+            string rootPath = Path.Combine(Environment.CurrentDirectory, "ScriptHostTests");
+
+            if (!Directory.Exists(rootPath))
+            {
+                Directory.CreateDirectory(rootPath);
+            }
+
+            _options = new ScriptApplicationHostOptions
+            {
+                ScriptPath = rootPath
+            };
+
+            _hostJsonFile = Path.Combine(rootPath, "host.json");
+            if (File.Exists(_hostJsonFile))
+            {
+                File.Delete(_hostJsonFile);
+            }
+        }
+
+        [Theory]
+        [InlineData(@"{
+                    'version': '2.0',
+                    }")]
+        [InlineData(@"{
+                    'version': '2.0',
+                    'extensionBundle': {
+                        'id': 'Microsoft.Azure.Functions.ExtensionBundle',
+                        'version': '(1.0.0,2.0.0]'
+                        }
+                    }")]
+        public void MissingOrValidExtenstionConfig_DoesNotThrowException(string hostJsonContent)
+        {
+            File.WriteAllText(_hostJsonFile, hostJsonContent);
+            Assert.True(File.Exists(_hostJsonFile));
+            var configuration = BuildHostJsonConfiguration();
+
+            ExtensionBundleOptionsSetup setup = new ExtensionBundleOptionsSetup(configuration);
+            ExtensionBundleOptions options = new ExtensionBundleOptions();
+            var ex = Record.Exception(() => setup.Configure(options));
+            Assert.Null(ex);
+        }
+
+        [Theory]
+        [InlineData(@"{
```

Comment:
Generally you only use a Theory if you're testing multiple scenarios. If you're just testing one (null) you can do this inline as a [Fact]